src/components/user-profile.component.js

```diff
@@ -86,8 +86,8 @@ export const UserProfile = ({
       <ImageZoom
         uri={{
           uri: initialUser.avatar_url
-            ? initialUser.avatar_url
-            : user.avatar_url,
+            ? `${initialUser.avatar_url}&cache=${initialUser.updated_at}`
```

Comment:
Changed to `lastModified`.
@lex111 keeping the value as it is, as it's easier to read for debugging purpose